src/BenchmarkDotNet/Toolchains/MonoWasm/WasmGenerator.cs

```diff
@@ -30,6 +35,8 @@ protected override void GenerateProject(BuildPartition buildPartition, Artifacts
                     .Replace("$COPIEDSETTINGS$", customProperties)
                     .Replace("$CONFIGURATIONNAME$", buildPartition.BuildConfiguration)
                     .Replace("$SDKNAME$", sdkName)
+                    .Replace("$RUNTIMEPACK$", CustomRuntimePack != null ? CustomRuntimePack : "")
```

Comment:
nit: could be simplified

```suggestion
                    .Replace("$RUNTIMEPACK$", CustomRuntimePack ?? "")
```